native/v-kit.sh

```diff
@@ -80,8 +80,21 @@ install_mezo() {
 }
 
 install_skip() {
+    # Empty version defaults to latest
+    if [[ -z "${CONNECT_VERSION}" ]]; then
+      CONNECT_VERSION="2.1.2"
```

Comment:
Just to make sure: We don't need the `v` prefix here, right?

Reply:
No, we don't need it ;)